src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Color {
+    RED,
+    BLUE,
+    BLACK,
+    WHITE,
+}
```

Comment:
It's a good practice to have an access modifier for the enum. Although it's not strictly necessary here since the default access modifier is 'package-private' which is often suitable for enums, explicitly declaring 'public' would make it clear that this enum is intended for use across multiple classes.